synapse/cortex.py

```diff
@@ -4888,6 +4888,13 @@ def getView(self, iden=None, user=None):
 
         return view
 
+    def reqView(self, iden):
+        view = self.getView(iden)
+        if view is None: # pragma: no cover
+            mesg = f'No view with iden: {self.valu.get("iden")}'
+            raise s_exc.NoSuchView(mesg=mesg, iden=self.valu.get('iden'))
```

Comment:
```suggestion
            mesg = f'No view with iden: {iden}'
            raise s_exc.NoSuchView(mesg=mesg, iden=iden)
```

I think `self.valu.get('iden')` is the cortex iden?

Reply:
it was a copy-pasta error from the View class in stormtypes. Good catch!